### DISCUSSION TITLE
I need help with a file

### DISCUSSION THREAD

**Discussion by MalteadaAIAI** [KadeArchive/Kade-Engine#310] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/83192833/116015912-3b485280-a5f8-11eb-9ba5-1cc1dcc2044a.png)
that appears to me when he tells me to change one thing
![image](https://user-images.githubusercontent.com/83192833/116015945-54510380-a5f8-11eb-8e82-b98ed91fac24.png)
Now I get this and I don't know what to do

**Comment by DizzyJune** [comment]
Would be nice if you listed said file? Otherwise we can't really help.

**Reply by MalteadaAIAI** [reply]
![image](https://user-images.githubusercontent.com/83192833/116095020-42f21080-a665-11eb-84a4-4956bb2a383a.png)

**Comment by puyoxyz** [comment]
send the full output

**Reply by MalteadaAIAI** [reply]
![image](https://user-images.githubusercontent.com/83192833/116835676-bbcefc00-ab80-11eb-84bc-3b1624ed05f4.png)

**Comment by puyoxyz** [comment, marked as answer]
The problem isn't with PlayState, but it's that you don't have Visual Studio 2017. Please read [this section of BUILDING.md](https://github.com/KadeDev/Kade-Engine/blob/master/BUILDING.md#windows-only-dependencies-only-for-building-to-windows-building-html5-on-windows-does-not-require-this) for more info.

**Reply by MalteadaAIAI** [reply]
Ok, I installed visual studio 2017 and the same

**Reply by puyoxyz** [reply]
> Ok, I installed visual studio 2017 and the same

OH GOD I MADE A TYPO I MEANT VISUAL STUDIO 2019 NOOOOOOOO SORRY SORRY SORRY SORRY

(so like uh uninstall 2017 and install visual studio 2019 but **make sure to follow the instructions in [BUILDING.md](https://github.com/KadeDev/Kade-Engine/blob/master/BUILDING.md#windows-only-dependencies-only-for-building-to-windows-building-html5-on-windows-does-not-require-this)**

**Reply by MalteadaAIAI** [reply]
No problem, I will try thx very much

**Reply by MalteadaAIAI** [reply]
![image](https://user-images.githubusercontent.com/83192833/116898773-050c6380-abf4-11eb-83f2-2a7a81d64119.png)
community right?

**Reply by puyoxyz** [reply]
yeah (and when you have the installer follow the instructions in [BUILDING.md](https://github.com/KadeDev/Kade-Engine/blob/master/BUILDING.md#windows-only-dependencies-only-for-building-to-windows-building-html5-on-windows-does-not-require-this)!!!!)

**Reply by MalteadaAIAI** [reply]
thx
